lib/libutils/isoc/arch/arm/setjmp_a64.S

```diff
@@ -51,6 +51,12 @@ setjmp:
 #define REG_ONE(REG1, OFFS)		str REG1, [x0, OFFS]
 	GPR_LAYOUT
 	FPR_LAYOUT
+#ifdef CFG_TA_FTRACE_SUPPORT
+	stp	x29, x30, [sp, #-16]!
+	mov	x29, sp
+	bl	ftrace_setjmp
+	ldp	x29, x30, [sp], #16
```

Comment:
`ftrace_setjmp()` can have changed a few more registers than these.

Reply:
Since we are already done with storing context registers in jump buffer (x0) before this call, so we have no argument to save and restore.

Reply:
OK